lib/stdlib/doc/src/supervisor.xml

```diff
@@ -534,38 +627,26 @@ child_spec() = #{id => child_id(),             % mandatory
     </func>
 
     <func>
-      <name name="start_child" arity="2" since=""/>
```

Comment:
I agree that current formulation is hard to read. However clause_i will not be possible to use with the new
documentation format from #8026 so we have to come up with an alternative solution.  clause_i is sometimes convenient but also quite error prone if the code changes it implementation adds/removes clauses.  I am all for splitting it into "most supervisors"  and  "simple one for one supervisors"

Reply:
I guessed as much when I saw #8026... This raises the question how this PR should be handled, though. Wait for #8026 to be merged and then rework it in the "new way"?

Reply:
I think we do not want add new things that will need to be change once #8026 is merged.  PR-8026 will not aim to be the final version, and other changes will be needed to make everything as we like it but we need to start somewhere.
Probably for now we will have to make some solution referring to if the function is called like x then otherwise if it is called like y then ...  I have a similar problem in ssl docs but have not yet had time to make a solution.

Reply:
No, what I mean is, how should I proceed with this PR in general? The most sensible way seems to be to "put it on hold" for now, wait for #8026 to be merged, then sort of write it again, this part and the others, according to the new way. This is ok for me, I don't mind :wink: WDYT?

Reply:
Sure if you feel that is easier its planed for master anyway. I will put a stalled label on it, and remove it once #8026 is merged.

Reply:
I think this is the best course, yes :)